### PR TITLE
Cleaned up integration-cli/docker_api_exec_test.go

### PR DESCRIPTION
**Pull request by ZJaffee**
As part of "Proposal: use of checkers on Integration test #16756"

### CODE REVIEW

**Comment by vdemeester** [comment]
@ZJaffee As stated by the build you need to format the modified files, with `gofmt -s -w`

File: integration-cli/docker_api_exec_test.go [hunk 1]
```diff
@@ -19,12 +20,10 @@ func (s *DockerSuite) TestExecApiCreateNoCmd(c *check.C) {
...
 	status, body, err := sockRequest("POST", fmt.Sprintf("/containers/%s/exec", name), map[string]interface{}{"Cmd": nil})
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusInternalServerError)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusInternalServerError)
 
-	if !bytes.Contains(body, []byte("No exec command specified")) {
-		c.Fatalf("Expected message when creating exec command with no Cmd specified")
-	}
+	c.Assert(body,checker.Not(checker.Contains),[]byte("No exec command specified"),Commentf("Expected message when creating exec command with no Cmd specified"))
```
Comment (vdemeester) [inline integration-cli/docker_api_exec_test.go]: Here and everywhere else on this file : s/Commentf/check.Commentf/

File: integration-cli/docker_api_exec_test.go [hunk 2]
```diff
@@ -33,20 +32,16 @@ func (s *DockerSuite) TestExecApiCreateNoValidContentType(c *check.C) {
 	dockerCmd(c, "run", "-d", "-t", "--name", name, "busybox", "/bin/sh")
 
 	jsonData := bytes.NewBuffer(nil)
-	if err := json.NewEncoder(jsonData).Encode(map[string]interface{}{"Cmd": nil}); err != nil {
-		c.Fatalf("Can not encode data to json %s", err)
-	}
+	c.Assert(json.NewEncoder(jsonData).Encode(map[string]interface{}{"Cmd": nil}), checker.IsNil,Commentf("Can not encode data to json"))
```
Comment (vdemeester) [inline integration-cli/docker_api_exec_test.go]: I would have split it into 2 lines to make it easier to read.

**Comment by HuKeping** [comment]
ping @ZJaffee

**Comment by calavera** [comment]
I'm very sorry but I'm going to close this PR until the errors are resolved. Please, feel free to send it again after testing that every test still works and the code is valid. `make test` and `make validate` are your friends.

As a rule of thumb, before opening all your PRs again, it will be easier to get them merged if you focus in one at a time, that way reviewers don't need to point the same errors over several changes.

**Pull request closed without merging**